deployment/src/main/java/io/quarkiverse/helm/deployment/QuarkusHelmWriterSessionListener.java

```diff
@@ -267,7 +268,8 @@ private Map<String, String> createValuesYaml(io.dekorate.helm.config.HelmChartCo
 
             // Create the values.<profile>.yaml file
             artifacts.putAll(writeFileAsYaml(mergeWithFileIfExists(inputDir, VALUES + YAML, toValuesMap(values)),
-                    getChartOutputDir(helmConfig, outputDir).resolve(VALUES + "." + profile + YAML)));
```

Comment:
@Sgitario I  believe the current behavior is that profile-specific values files are generated with as values"."{profile}. Even if the documentation said otherwise. That is the reason I also introduced a change in ValueReferenceConfig.java.